### PR TITLE
initial snippets for classroom course management

### PR DESCRIPTION
**Pull request by lesley2958**

### CODE REVIEW

File: classroom/snippets/createCourse.gs [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+// [START classroom_create_course]
+/**
+ * Creates 10th Grade Biology Course.
+ */
+function createCourse() {
+  var course = {
+    'name': '10th Grade Biology',
```
Comment (grant) [inline classroom/snippets/createCourse.gs]: Keys should not have quotes.

File: classroom/snippets/getCourse.gs [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+// [START classroom_get_course]
+/**
+ * Lists all course names and ids. 
+ */
+
```
Comment (grant) [inline classroom/snippets/getCourse.gs]: No newline here.

File: classroom/snippets/getCourse.gs [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+function getCourse() {
+  var courseId = '123456';
+  try {
+    var course = Classroom.Courses.get(courseId);
+    Logger.log('Course \"%s\" found. ', course.name);
```
Comment (grant) [inline classroom/snippets/getCourse.gs]: No need to escape `"`.

File: classroom/snippets/listCourses.gs [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+  while (true) {
+    var response = Classroom.Courses.list(optionalArgs);
+    var courses = response.courses;
+    if (!pageToken) {
+       break;
+    }
+  }
+  if (courses.length == 0) {
+    Logger.log("No courses found.");
+  }
```
Comment (grant) [inline classroom/snippets/listCourses.gs]: `} else {`

File: classroom/snippets/listCourses.gs [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+    pageSize: 100
+  };
+  while (true) {
+    var response = Classroom.Courses.list(optionalArgs);
+    var courses = response.courses;
+    if (!pageToken) {
+       break;
+    }
+  }
+  if (courses.length == 0) {
```
Comment (grant) [inline classroom/snippets/listCourses.gs]: `if (courses.length) {`

Comment (lesley2958) [reply classroom/snippets/listCourses.gs]: why would it be `if (courses.length)`? `courses` is declared as an empty list above so it should always exist regardless of whether there are courses or not for the user

**Comment by lesley2958** [comment]
added changes for all feedback except the `if (courses.length)`.

File: classroom/snippets/listCourses.gs (hunk 5 above)
Comment (grant) [reply classroom/snippets/listCourses.gs]: You could either do:
```js
if (course.length === 0) {
// or
if (course.length) {
```

I tend to prefer shorter statements, but we are currently not consistent about this, so it's fine if you'd like it either of the above options.

Examples:
https://github.com/gsuitedevs/apps-script-samples/search?q=length%29&unscoped_q=length%29
https://www.quora.com/How-do-you-check-if-an-array-is-empty-in-JavaScript#wxx6Nk378

**Comment by grant** [comment]
Ping after fixing the `===` statements and we'll merge.

**Comment by lesley2958** [comment]
> Ping after fixing the `==` (to `===`) statements and we'll merge.

great, thanks! submitted the change

**Review by TheMapotakes — COMMENTED** [review]
I dit it

**Pull request merged**